### PR TITLE
feat: add channel

### LINKED ISSUE #182: Сделать добавление канала

### PR DESCRIPTION
**Pull request by olliva**
closes #182

### CODE REVIEW

File: app/actions/channels.js [hunk 1]
```diff
@@ -15,10 +22,22 @@ export function removeChannel(channelId) {
...
     payload: data,
   };
 }
+
+export function replaceDirtyChannel(channel) {
+  return {
+    type: A.REPLACE_DIRTY_CHANNEL,
+    channel,
+  };
+}
```
Comment (SeqviriouM) [inline app/actions/channels.js]: will be better 
```{
type: A.REPLACE_DIRTY_CHANNEL,
payload: channel
}```

File: app/components/ChannelItem/index.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default class ChannelItem extends React.Component {
 
 
   setChannel = () => {
-    this.props.setCurrentChannel(this.state.id);
+    this.props.setCurrentChannel(this.props.item.get('id'));
```
Comment (SeqviriouM) [inline app/components/ChannelItem/index.jsx]: why replace old option?

File: app/components/NewChannelItem/index.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+
+  componentDidMount = () => {
+    const newChannelItem = this.refs.newChannel.getDOMNode();
+    const newChannelInput = this.refs.newChannelInput.getDOMNode();
+    setTimeout(function timeout() {
+      newChannelItem.className = newChannelItem.className + ' new-channel_animated';
+      newChannelInput.focus();
+    }, 1);
```
Comment (SeqviriouM) [inline app/components/NewChannelItem/index.jsx]: Why we need use setTimeout here?May be then move this cocde to another place?

Comment (Nitive) [inline app/components/NewChannelItem/index.jsx]: @olliva change DOM by using state please

Comment (Nitive) [inline app/components/NewChannelItem/index.jsx]: And we can use react-motion to do animation easy (use willEnter and willLeave props)

**Comment by Nitive** [comment]
jquery-react

**Pull request merged**